content/en/blog/falcosidekick-reponse-engine-part-6-cloud-run.md

```diff
@@ -0,0 +1,188 @@
+
+Note that we're going to use the ko tool to build and push our container image which is created by Google. ko is a simple and fast container image builder for Go applications.
+
+To learn more, see the [official repository](https://github.com/google/ko) of the project.
+
+We are also going to use Container Registry as an image repository service provided by the Google Cloud to store, manage, and secure your Docker container images. Alternatively, you can also use DockerHub, quay.io, etc.
+
+```shell
+$ git clone https://github.com/developer-guy/kubernetes-response-engine-based-on-gke-and-cloud-run.git
+$ cd kubernetes-response-engine-based-on-gke-and-cloud-run
+$ GKE_CLUSTER_NAME=$(kubectl config view --minify -o jsonpath='{.clusters[].name}{"\n"}')
+$ PROJECT_ID=$(gcloud config get-value project)
+$ FUNCTION_NAME=pod-deleter
+$ export KO_DOCKER_REPO=gcr.io/$PROJECT_ID # Please, change this variable if you are not using Container Registry.
+$ gcloud config set run/region us-west1
+$ gcloud config set run/platform managed
+$ gcloud run deploy $FUNCTION_NAME --image=$(ko publish .) \
+    --set-env-vars GKE_CLUSTER_NAME=$GKE_CLUSTER_NAME \
+    --set-env-vars PROJECT_ID=$PROJECT_ID
+...
+Allow unauthenticated invocations to [pod-deleter] (y/N)?  N
+
+Deploying container to Cloud Run service [pod-deleter] in project [developerguy-311909] region [us-west1]
+✓ Deploying... Done.
+  ✓ Creating Revision...
+  ✓ Routing traffic...
+Done.
+Service [pod-deleter] revision [pod-deleter-00002-cej] has been deployed and is serving 100 percent of traffic.
+Service URL: https://pod-deleter-uoz6q2wria-uw.a.run.app
+```
+
+### Install Falco + Falcosidekick
+
+Now, it is time to set up `Falco`, `Falcosidekick` with the `Cloud Run` output type enabled.
+
+```shell
+$ kubectl create namespace falco
+$ helm repo add falcosecurity https://falcosecurity.github.io/charts
+$ helm repo update
+$ helm install falco falcosecurity/falco --namespace falco \
+--set ebpf.enabled=true \
+--set falcosidekick.enabled=true \
+--set falcosidekick.webui.enabled=true \
+--set falcosidekick.config.gcp.cloudrun.endpoint=$(gcloud run services list --format json | jq -r ".[] | select(.metadata.name==\"$FUNCTION_NAME\") | .status.address.url") \
+--set falcosidekick.config.gcp.cloudrun.jwt=$(gcloud auth print-identity-token)
+```
+
+Check the logs to see if `Cloud Run` output enabled for Falcosidekick.
+
+```shell
+$ kubectl logs -f falco-falcosidekick-7cd7bc6859-2nd9t --namespace falco
+falco-falcosidekick-7cd7bc6859-2nd9t falcosidekick 2021/06/07 16:03:14 [INFO]  : Enabled Outputs : [GCPCloudRun WebUI]
+falco-falcosidekick-7cd7bc6859-2nd9t falcosidekick 2021/06/07 16:03:14 [INFO]  : Falco Sidekick is up and listening on :2801
+...
+```
+
+If you see the `GCPCloudRun` in the list of enabled outputs, you consider that everything is good 👍.
```

Comment:
If you see the `GCPCloudRun` in the list of enabled outputs, you can confirm that everything is working as expected 👍.

Reply:
thank you so much, done.

Reply:
thank you so much, done.